app-games/src/main/java/com/aptoide/android/aptoidegames/feature_payments/credit_card/AdyenCreditCardScreen.kt

```diff
@@ -369,22 +370,33 @@ private fun AdyenCreditCardScreenPortrait(
 private fun AdyenCreditCardView(
   cardComponent: CardComponent,
   modifier: Modifier = Modifier,
+  delayMillis: Long = 100L
 ) {
   val lifecycleOwner = LocalLifecycleOwner.current
+  var isVisible by remember { mutableStateOf(false) }
+
+  LaunchedEffect(Unit) {
+    delay(delayMillis)
```

Comment:
I don't like relying on a delay because different devices may take different time to process the same thing.
But if there is no other solution, then it's ok I think.